app/models/email.rb

```diff
@@ -11,7 +9,6 @@ def self.build_from_params(params)
     renderer = EmailRenderer.new(params: params)
 
     new(
-      notification_id: params[:notification_id],
```

Comment:
Nice! 🎉 

I just thought... we might have to move this tiny change into a separate PR and deploy that first to prevent an old version of the application from trying to save this field - alternatively, since the errors will be sent to Sentry and we don't really care about the new system yet, we can leave it.